diff --git a/node_modules/ejs/lib/ejs.js b/node_modules/ejs/lib/ejs.js
index 65590ea..ff8d7b2 100755
--- a/node_modules/ejs/lib/ejs.js
+++ b/node_modules/ejs/lib/ejs.js
@@ -632,9 +632,28 @@ Template.prototype = {
     else {
       src = this.source;
     }
+    const str=`
+    var _ENCODE_HTML_RULES = {
+      '&': '&amp;',
+      '<': '&lt;',
+      '>': '&gt;',
+      '"': '&#34;',
+      "'": '&#39;'
+    };
+    var _MATCH_HTML = /[&<>'"]/g;
     
+    function encode_char(c) {
+      return _ENCODE_HTML_RULES[c] || c;
+    }
+    exports.escapeXML = function (markup) {
+      return markup == undefined
+        ? ''
+        : String(markup)
+          .replace(_MATCH_HTML, encode_char);
+    };
+    `
     if (opts.client) {
-      src = 'escapeFn = escapeFn || ' + escapeFn.toString() + ';' + '\n' + src;
+      src = 'escapeFn = escapeFn || ' + str + ';' + '\n' + src;
       if (opts.compileDebug) {
         src = 'rethrow = rethrow || ' + rethrow.toString() + ';' + '\n' + src;
       }
